bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/InboxResource.java

```diff
@@ -108,6 +108,13 @@ public Response ignore(@PathParam("thingUID") String thingUID) {
         inbox.setFlag(new ThingUID(thingUID), DiscoveryResultFlag.IGNORED);
         return Response.ok().build();
     }
+    
+    @POST
+    @Path("/unignore/{thingUID}")
```

Comment:
Yes, I thought the same, but the endpoint for "ignore" also is "@Path("/ignore/{thingUID}")" (see the lines above). Maybe we should take the time to recheck the resource pathes. Is there any other application, which uses the new resources already? @cdjackson Does HABmin?